droopler.info.yml

```diff
@@ -18,6 +18,7 @@ dependencies:
   - content_translation
   - contextual
   - d_content
+  - d_demo
```

Comment:
Czy tu się coś nie podzieje użytkownikom, którzy już mają Drooplera i go aktualizują? Może lepiej byłoby włączyć to demo już w install tasku